support/android/build-apk/src/main.rs

```diff
@@ -90,14 +90,18 @@ fn main() {
         process::exit(1);
     }
 
+    // Retrieve the Android SDK version target env variable
+    let default_platform = "android-18".to_owned();
+    let android_platform = env::var("ANDROID_PLATFORM").unwrap_or(default_platform);
```

Comment:
Let's make passing the environment variable mandatory, just like the other ANDROID_* variables (see the top of main for examples).